src/frontend/dsl/mod.rs

```diff
@@ -172,6 +162,27 @@ impl<F, TraceArgs> CircuitContext<F, TraceArgs> {
     }
 }
 
+impl<F: Field + Hash, TraceArgs> CircuitContext<F, TraceArgs> {
+    /// Sets the fixed generation function for the circuit. The fixed generation function is
```

Comment:
@qwang98 this does not "set", but "executes" and sets the assignations

Reply:
Fixed.